cluster/gce/gci/configure-helper.sh

```diff
@@ -1269,6 +1269,10 @@ EOF
 function create-kubeconfig {
   local component=$1
   local token=$2
+
+  if [[ -e "${KUBE_HOME}/bin/gke-internal-configure-helper.sh" ]]; then
+    gke-internal-create-kubeconfig ${component} ${token}
+  else
```

Comment:
Nit: please indent the now-`else` clause.

Reply:
Thanks. Fixed the indentation.